src/libs/OptionsListUtils.ts

```diff
@@ -487,19 +580,19 @@ function createOption(accountIDs, personalDetails, report, reportActions = {}, {
         isPolicyExpenseChat: false,
         isOwnPolicyExpenseChat: false,
         isExpenseReport: false,
-        policyID: null,
+        policyID: undefined,
         isOptimisticPersonalDetail: false,
     };
 
     const personalDetailMap = getPersonalDetailsForAccountIDs(accountIDs, personalDetails);
-    const personalDetailList = _.values(personalDetailMap);
-    const personalDetail = personalDetailList[0] || {};
+    const personalDetailList = Object.values(personalDetailMap).filter((details): details is PersonalDetails => !!details);
+    const personalDetail = personalDetailList[0];
```

Comment:
Hi @ntdiary !
I think this https://github.com/Expensify/App/blob/7962905e107ce99b7ec9a18dc76fd3b70ba71e9c/src/libs/OptionsListUtils.ts#L588 is filterning falsy values from `personalDetailList` right?  so `personalDetail` should always be defined in my opinion. 
Is the crash already happend, or this was just a assumption?

Reply:
hmm its weird because I even run this is TS playground and it should filter those falsy values 
<img width="2011" alt="image" src="https://github.com/Expensify/App/assets/25230417/9d6ba938-8a4a-4100-a57b-c7d8000bbc38">
but yeah I will make a PR with a fix.

Reply:
@ntdiary do you know if there is an issue for that created?

Reply:
Let's fix this asap. This blocks testing important parts of the app.
This happens because `const personalDetailMap = getPersonalDetailsForAccountIDs(accountIDs, personalDetails);` can be empty json `{}`

https://github.com/Expensify/App/assets/108292595/1d2a8cf0-a2df-4437-b504-4fbd0b454750